.github/workflows/maven.yml

```diff
@@ -34,6 +34,14 @@ jobs:
     - name: Build with Maven
       run: mvn -B package --file pom.xml
 
+    # Copy the files to the gh-pages branch
```

Comment:
Should this really be executed for a PR? It would seem odd that each PR run updates what is displayed on the live website.
Also, as `EisopSiteGenerator` isn't executed, the templates and releases aren't updated.
So maybe this should go somewhere else?

Reply:
Thanks! From the log it looks like this isn't working correctly. I've filed #11 for follow-up work.